large_media

```diff
@@ -1 +1 @@
-Subproject commit c946e65a245948f384835f05a88e6b9a13a1a090
+Subproject commit e683e3dcf9540cd17340984b6dadd5459687af16
```

Comment:
**Caution!** This contains a submodule update

Reply:
Yes, indeed. For the animated gif that I also added to the doc page.